packages/react-ui-core/src/DateInput/__tests__/DateInput-test.js

```diff
@@ -5,16 +5,16 @@ import DateInput from '../DateInput'
 describe('DateInput', () => {
   it('renders a DateInput', () => {
     const wrapper = shallow(<DateInput />)
-    expect(wrapper.find('DatePicker')).toHaveLength(1)
+    expect(wrapper.html().includes('DatePicker')).toEqual(true)
   })
 
   it('passes through props', () => {
-    const wrapper = shallow(<DateInput id="foo" />)
-    expect(wrapper.find('DatePicker').prop('id')).toBe('foo')
+    const wrapper = shallow(<DateInput datepickerId="foo" />)
+    expect(wrapper.html().includes('id="foo"')).toEqual(true)
   })
 
   it('applies a custom className', () => {
     const wrapper = shallow(<DateInput className="DatePicker" />)
-    expect(wrapper.find('DatePicker').prop('className')).toBe('DatePicker')
+    expect(wrapper.html().includes('class="DatePicker"')).toEqual(true)
```

Comment:
It looks like the code now hard-codes `DatePicker` classname rather than using the `className` prop, so though this test might be passing it doesn't look like a valid test. Remove the test if we're not supporting custom `className` prop, or modify `DateInput` so it applies the `className` prop.

Reply:
I need to update that, that was supposed to use className as the main option, will fix